translation-client/src/main/java/eu/europeana/api/translation/client/utils/TranslationClientUtils.java

```diff
@@ -1,30 +1,107 @@
 package eu.europeana.api.translation.client.utils;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import eu.europeana.api.translation.client.exception.TranslationApiException;
+import eu.europeana.api.translation.definitions.language.LanguagePair;
+import org.apache.commons.lang3.StringUtils;
 import org.springframework.web.util.UriBuilder;
 
 import java.net.URI;
+import java.util.*;
 import java.util.function.Function;
 
 public class TranslationClientUtils {
 
+    private static final ObjectMapper mapper = new ObjectMapper();
+
     public static final String TRANSLATE_URL = "translate";
     public static final String LANG_DETECT_URL = "detect";
+    public static final String INFO_ENDPOINT_URL = "actuator/info";
+
+    // info endpoint constants
+    public static final String CONFIG = "config";
+    public static final String DETECT = "detect";
+    public static final String TRANSLATE = "translate";
+    public static final String SUPPORTED = "supported";
+    public static final String SOURCE = "source";
+    public static final String TARGET = "target";
 
     private TranslationClientUtils() {
 
     }
 
     /**
      * Builds the url path based on the path string passed
+     *
      * @param path
      * @return
      */
     public static Function<UriBuilder, URI> buildUrl(String path) {
         return uriBuilder -> {
             UriBuilder builder =
                     uriBuilder
-                            .path("/" + path) ;
+                            .path("/" + path);
             return builder.build();
         };
     }
+
+    private static JsonNode getConfigNode(String json) throws TranslationApiException {
+        if (!StringUtils.isEmpty(json)) {
```

Comment:
there is a more efficient way to extract the required JsonNode only, which will simplify significantly this code: 
https://cassiomolin.com/programming/using-jackson-and-json-pointer-to-query-and-parse-an-arbitrary-json-node/ 

would be nice to update the implementation to use this approach

ObjectMapper mapper = new ObjectMapper();
JsonNode node = mapper.readTree(json);
JsonNode coordinatesNode = node.at("/address/coordinates");